src/App.css

```diff
@@ -1,39 +1,56 @@
-.App {
-  text-align: center;
+/* .wrapper {
+  width: 340px;
+  height: 540px;
+  padding: 10px;
+  border-radius: 10px;
+  background-color: #485461;
+  background-image: linear-gradient(315deg, #485461 0%, #28313b 74%);
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.display {
+  height: 100px;
+  width: 100%;
+  margin-bottom: 10px;
+  padding: 0 10px;
+  background-color: #465c6e2d;
+  border-radius: 10px;
   display: flex;
-  flex-direction: column;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
+  justify-content: flex-end;
   color: white;
+  font-weight: bold;
+  box-sizing: border-box;
 }
 
-.App-link {
-  color: #61dafb;
+.buttonPanel {
+  width: 100%;
+  height: calc(100% - 110px);
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(5, 1fr);
+  grid-gap: 10px;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+button {
+  border: none;
+  background-color: rgb(80, 60, 209);
+  font-size: 24px;
+  color: rgb(255, 255, 255);
+  font-weight: bold;
+  cursor: pointer;
+  border-radius: 10px;
+  outline: none;
+}
+
+button:hover {
+  background-color: rgb(61, 43, 184);
+}
 
-  to {
-    transform: rotate(360deg);
-  }
+.equals {
+  grid-column: 3 / 5;
+  background-color: rgb(243, 61, 29);
 }
+
+.equals:hover {
+  background-color: rgb(228, 39, 15);
+} */
```

Comment:
- [ ] As mentioned by the last reviewer, please clean up commented codes from your codebase. All the codes in this file have been commented out. Please, let's clean it up. :pray: You may delete the file altogether if its not useful.